client/blocks/app-banner/index.jsx

```diff
@@ -1,9 +1,11 @@
+import config from '@automattic/calypso-config';
 import { Button, Card } from '@automattic/components';
 import classNames from 'classnames';
-import { localize } from 'i18n-calypso';
+import { localize, useRtl } from 'i18n-calypso';
 import { get } from 'lodash';
+import lottie from 'lottie-web/build/player/lottie_light';
```

Comment:
Do we use lottie for every single instance of `AppBanner`? If not, we might gain some performance by async loading it separately for just the instances which need it.

Reply:
We will eventually (hopefully in a matter of weeks) be using the icon animation in all instances of the banner, so don't think there will be much gain from this, but let me know if you think otherwise.

Reply:
That sounds fair to me. :)